### DISCUSSION TITLE
Global pending component

### DISCUSSION THREAD

**Discussion by maksymskuibida** [TanStack/router#1612] [Q&A]
Hello everyone. I have a question. Somewhy I thought that if I set pendingComponent in __root.tsx, it will be default pending component for every route, but it is not. Is there any way I can set default pending component for everypage and it will appear, instead of white screen if some route is pending?

**Comment by schiller-manuel** [comment]
are you looking for https://tanstack.com/router/v1/docs/framework/react/api/router/RouterOptionsType#defaultpendingcomponent-property ?

**Reply by maksymskuibida** [reply]
Wow, yes! I searched exactly "defaultPendingComponent" in docs search because I believed I saw it earlier in docs, but I found nothing. Thank you so much.


It is impossible to find it in docs by any of "defaultPending", "default pending" and even "pending". It looks like it is not indexed in search, I will file an issue about this
